storage/innobase/include/trx0purge.h

```diff
@@ -438,6 +445,11 @@ class purge_sys_t
   @param	already_stopped	True indicates purge threads were
 				already stopped */
   void stop_FTS(const dict_table_t &table, bool already_stopped=false);
+
+  /** Cleanse purge queue to remove the rseg that reside in undo-tablespace
+  marked for truncate.
+  @param space undo tablespace being truncated */
+  void cleanse_purge_queue(const fil_space_t& space);
```

Comment:
In the common formatting style, there should be a space *before* a reference or pointer marker, not after it. It was the opposite way around in the original InnoDB formatting style.